test/AspNet.Security.OAuth.Providers.Tests/Huawei/bundle.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/justeat/httpclient-interception/master/src/HttpClientInterception/Bundles/http-request-bundle-schema.json",
+  "items": [
+    {
+      "comment": "https://developer.huawei.com/consumer/cn/doc/development/HMSCore-References/account-obtain-token_hms_reference-0000001050048618",
+      "uri": "https://oauth-login.cloud.huawei.com/oauth2/v3/token",
+      "method": "POST",
+      "contentFormat": "json",
+      "contentJson": {
+        "access_token": "secret-access-token",
+        "refresh_token": "secret-refresh-token",
+        "expires_in": 3600,
+        "id_token": "secret-id-token",
+        "scope": "openid profile",
+        "token_type": "Bearer"
+      }
+    },
+    {
+      "comment": "https://developer.huawei.com/consumer/cn/doc/development/HMSCore-References/get-user-info-0000001060261938",
+      "uri": "https://account.cloud.huawei.com/rest.php?nsp_svc=GOpen.User.getInfo",
```

Comment:
Reading the documentation, there's an optional body parameter to get the nickname in the response. It would be worth adding functionality to support that, otherwise a user needs to override quite a lot of things to get the request to do that if they want/need it.